Source/gendung.h

```diff
@@ -1,4 +1,8 @@
-//HEADER_GOES_HERE
+/**
+ * @file gendung.h
+ *
+ * Interface of general dungion generation code.

```

Comment:
```suggestion
 * Interface of general dungeon generation code.
```

Apply same fix to other places too.